### ISSUE TITLE
Curriculum

### ISSUE THREAD

**Issue by chynnasevilleno** [chynnasevilleno/SAD-Enrollment#6]
Currently, we have a school year input field sa curriculum but it's confusing me.

IF a user creates a unique curriculum, say 2018. This curriculum can be reused by SY 2018-2019 until 2019-2020 if DepEd chooses to do so. I believe this curriculum option should be adopted sa School Year initialization? Idk? What do you think @aaajii though?

**Comment by coleenyuson** [comment]
yes chy.. pwede talaga. hindi man sila yearly nagachange ng curriculum. pero memo man ata to by the DepEd

**Comment by aaajii** [comment]
Yes, it should be. Adding 'School Year' attribute to the Curriculum model would make the data very dynamic. I'd say we create an entity named "SchoolYear_Curriculums" where a Curriculum may use many School Year instances and vice versa.
